controller.py

```diff
@@ -73,6 +76,66 @@ def authenticated_client_from_request(_db, required=True):
     return INVALID_CREDENTIALS
 
 
+class IndexController(object):
+
+    def __init__(self, _db):
+        self._db = _db
+
+    def opds_catalog(self):
+        url = ConfigurationSetting.sitewide(self._db, Configuration.BASE_URL_KEY).value
+        catalog = dict(
+            id=url,
+            title='Library Simplified Metadata Wrangler',
+        )
+
+        catalog['links'] = [
```

Comment:
This hardcoding... 🙈, but dynamically loading the links would have been about the same, and I wasn't that the data structure to hold this information wouldn't look pretty much exactly like this. I'm open to alternatives.

Reply:
One advantage of loading the links with `url_for` is you can make them absolute URLs, which will be easier for a client to handle. Another is the test that generates this file will break if we change anything about the controllers. If it's possible to use `url_for` to make URI Templates then I think you should go for it. But it might not work well.